frontend/src/metabase/parameters/components/ValuesSourceSettings/ValuesSourceSettings.unit.spec.tsx

```diff
@@ -48,4 +48,46 @@ describe("ValuesSourceSettings", () => {
       values: ["A"],
     });
   });
+
+  it("Editing the values source should be disabled when the filter has linked filters", () => {
```

Comment:
we usually don't sentence-case test names, it should be `editing the values ...`